### ISSUE TITLE
Various color issues of dark high-contrast color scheme

### ISSUE THREAD

**Issue by myovchev** [equinusocio/vsc-material-theme#173]
First congrats for the great work, a good bit of my daily work joy is brought by your theme!

I'd like to bring up an issue (IMO) with the comment color of high contrast dark variation of the VSC theme - it feels too dark and is not readable, one idea lighten grey should do the work.
Another UI improvement would be to better colorize the current line marker, it's also mostly non-visible.

I think the same issues apply for the default (non high-contrast) version of the Dark scheme but I believe that's why we have high-contrast variation :)

**Comment by snwflake** [comment]
That was my problem as well.
If you have the marketplace install, you can simply bring up the theme in `~/.vscode/extensions/Equinusocio.vsc-material-theme-2.0.1/themes/Material-Theme-Darker-High-Contrast.json` and set the comment color directly. 

```json
{
    "name": "Comment",
    "scope": [
        "punctuation.definition.comment",
        "comment.block",
        "comment.line",
        "comment.block.documentation"
    ],
    "settings": {
        "fontStyle": "italic",
        "foreground": "#608b4e88"
    }
},
```

(That's the default "comment"-green, with lower opacity)
The reason I hardcoded that, was that if I changed the setting via 

```json
"editor.tokenColorCustomizations": {
    "[Material Theme Darker High Contrast]": {
        "comments": "#608b4e"
    },
},
```

Forward slashes and block slashes would stay grey (I honestly don't know, nor care why).

As for the current line marker, I found that the following works for me

```json
"workbench.colorCustomizations": {
    "editor.lineHighlightBackground": "#2f3333",
    "editorCursor.foreground": "#ffff22",
    "editor.selectionBackground": "#85854155",
},
```

(I threw in bonus with the text selection)

**Comment by equinusocio** [comment]
Hi @snwflake, with vs code you can override all the theme elements. I suggest you to check this doc section https://code.visualstudio.com/docs/getstarted/themes#_customizing-a-color-theme.

## TL;DR
Put this on your settings file:

```js
    "editor.tokenColorCustomizations": {
        "[Material Theme Ocean]": {
            "textMateRules": [
                {
                    "scope": [
                        "punctuation.definition.comment",
                        "comment.block",
                        "comment.line",
                        "comment.block.documentation"
                    ],
                    "settings": {
                        "foreground": "#FF0000"
                    }
                }
            ]
        },
    },
```

**Comment by snwflake** [comment]
Ahh, so those are textMateRules. Perfect!
I was too lazy googling, since I resolved it within a minute.

Thank you for the info!

**Comment by myovchev** [comment]
Thanks for your input - both of you.

**Comment by myovchev** [comment]
@equinusocio Do you by any chance have an idea what is the scope of currently active line background. I have searched the color reference docs of vscode and haven't find it. The comments color override works like a charm, thanks a lot for the tip.

**Comment by scottschup** [comment]
I've found this same answer in several places, but for whatever reason, it just wasn't working for me with any of the "Material Theme *" themes. No matter what I did, the `//` in JS and first `#` in Ruby/Shell won't change colors. 

To get around this, I had to use the "Community" version of the "Material Theme" themes: e.g. "Community Material Theme Darker High Contrast".
